### PR TITLE
fix octobot stop method

### PR DESCRIPTION
**Pull request by GuillaumeDSM**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -54,9 +54,9 @@ jobs:
       {
         $Env:TENTACLES_URL_SUBCATEGORY = ""
         If ($env:build_sourceBranch -Contains "refs/tags/") {
-          $Env:TENTACLES_URL_TAG = "latest"
-        } else {
           $Env:TENTACLES_URL_TAG = ""
+        } else {
+          $Env:TENTACLES_URL_TAG = "latest"
```
Comment (Herklos) [inline azure-pipelines.yml:59]: :+1:

File: tests/functional_tests/launch_test/launch_test.py [hunk 2]
```diff
@@ -33,4 +35,27 @@ async def test_run_bot():
     bot.task_manager.init_async_loop()
     await start_bot(bot, init_logger())
     await asyncio.sleep(10)
-    bot.task_manager.stop_tasks()
```
Comment (Herklos) [inline tests/functional_tests/launch_test/launch_test.py:36]: Why don't we use `stop_tasks()` anymore ?

Comment (GuillaumeDSM) [reply tests/functional_tests/launch_test/launch_test.py:36]: to be able to await the stop (stop tasks is not a coroutine and is using call_soon which makes it not awaitable)

Comment (Herklos) [reply tests/functional_tests/launch_test/launch_test.py:36]: Why not just calling `task_manager.stop_tasks` in `async def stop_bot(bot)`  ?

**Pull request merged**